packages/react-native/scripts/react-native-xcode.sh

```diff
@@ -176,7 +178,7 @@ else
   if [[ $EMIT_SOURCEMAP == true ]]; then
     EXTRA_COMPILER_ARGS="$EXTRA_COMPILER_ARGS -output-source-map"
   fi
-  "$HERMES_CLI_PATH" -emit-binary $EXTRA_COMPILER_ARGS -out "$DEST/main.jsbundle" "$BUNDLE_FILE"
+  "$HERMES_CLI_PATH" -emit-binary $EXTRA_COMPILER_ARGS $HERMES_EXTRA_ARGS -out "$DEST/main.jsbundle" "$BUNDLE_FILE"
```

Comment:
Could you add the `"` around the variables? Otherwise our linter will complain. 🙄 
Thank you so much!

```suggestion
  "$HERMES_CLI_PATH" -emit-binary "$EXTRA_COMPILER_ARGS" "$HERMES_EXTRA_ARGS" -out "$DEST/main.jsbundle" "$BUNDLE_FILE"
```